tools/shared.py

```diff
@@ -165,12 +165,23 @@ def run_process(cmd, universal_newlines=True, check=True, *args, **kw):
 
 def check_call(cmd, *args, **kw):
   try:
-    run_process(cmd, *args, **kw)
+    proc = run_process(cmd, *args, **kw)
     logging.debug('Successfully executed %s' % ' '.join(cmd))
+    return proc
   except subprocess.CalledProcessError as e:
     exit_with_error("'%s' failed (%d)", ' '.join(cmd), e.returncode)
 
 
+def run_process_js(filename, engine, jsargs=[], *args, **kw):
+  """Execute javascript tool using run_process.
```

Comment:
"using run_process" seems very implementation-detail-y to me, does this distinction matter to callers?